ledfx/config.py

```
@@ -869,6 +870,12 @@ def save_config(config: dict, config_dir: str) -> None:
         None
     """
     config_file = ensure_config_file(config_dir)
+    # Ensure logger is initialized (tests may call save_config without module init)
+    try:
+        _LOGGER
```

Ruff (0.13.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)